src/OpenLoco/src/Objects/VehicleObject.h

```diff
@@ -111,7 +111,7 @@ namespace OpenLoco
         uint8_t frontBogieSpriteInd; // 0x02 index of bogieSprites struct
         uint8_t backBogieSpriteInd;  // 0x03 index of bogieSprites struct
         uint8_t bodySpriteInd;       // 0x04 index of a bodySprites struct
-        uint8_t var_05;
+        uint8_t emitterHorizontalPos;
```

Comment:
strange the horizontal and vertical components are in different structs. re. naming, maybe `exhaustEmitterX` and `exhaustEmitterY`?

Reply:
OpenLoco uses Z-Up no?

Reply:
ah yeah I think you're right, but is this in screenspace or worldspace here?

Reply:
It's in car space, which is closer to world space

Reply:
Its in its own special world in terms of units. Its a position + 128. CS has done this as he wanted 0 to be a valid value but also wanted some value for invalid. So he chose 0 as invalid and then added 128 to all the values. So its basically of the range -127 to +127 but translated to 1-255.